app/views/customers/show.html.erb

```diff
@@ -36,9 +36,13 @@
   <% end %>
 
   <h2 class="center">Account Information</h2>
-  <div class="row" style="height:200px;">
+  <div class="row">
     <div class="col s6">
       <h5 class="center">Contact Info</h5>
+      <p class="center"><b style="padding-right:1em">Name:</b><%= current_customer.first_name %> <%= current_customer.last_name %></p>
+      <p class="center"><b style="padding-right:1em">Adress:</b><%= current_customer.address %></p>
+      <p class="center"><b style="padding-right:1em">City:</b><%= current_customer.city %> </p>
+      <p class="center"><b style="padding-right:1em">Adress 2:</b><%= current_customer.Address2 %></p>
```

Comment:
@cdanmar This looks good! One question. What happens if the fields are `nil`?

Reply:
To answer your question, the line will display blank (it will not error out). We can make additional cosmetic changes if needed.